src/cryptoadvance/specter/wallet.py

```diff
@@ -1740,14 +1744,16 @@ def createpsbt(
                 True,  # bip32-der
             )
 
-            b64psbt = r["psbt"]
+            # Always explicitly fill psbt with any missing fields
```

Comment:
This can create unnecessary large QR codes. In general, taproot_derivations should only be set to True if the device can support it.